systemtests/src/test/java/io/enmasse/systemtest/isolated/monitoring/MonitoringTest.java

```diff
@@ -150,22 +161,29 @@ void testAddressQueries() throws Exception {
                 .build();
 
         resourcesManager.setAddresses(false, topic, queue);
+        String infraUuid = addressSpace.getAnnotation(AnnotationKeys.INFRA_UUID);
+        System.out.println("Infra UUID::" + infraUuid);
```

Comment:
please use LOGGER instead of System.out.println

Reply:
:+1: Fixed